homeassistant/components/soma/strings.json

```diff
@@ -16,8 +16,10 @@
           "host": "[%key:common::config_flow::data::host%]",
           "port": "[%key:common::config_flow::data::port%]"
         },
-        "description": "Please enter connection settings of your SOMA Connect.",
-        "title": "SOMA Connect"
```

Comment:
What is the difference here specifying `description` where on other places `title` is used?

Reply:
If I understood correctly, the title is populated automatically. So I thought if I remove it, the title is the same for all integrations. I did not want to remove the content, so moved that into the description.